src/test/java/bridge/service/BridgeGameTest.java

```diff
@@ -0,0 +1,163 @@
+package bridge.service;
+
+import bridge.BridgeMaker;
+import bridge.BridgeNumberGenerator;
+import bridge.domain.BridgeMap;
+import bridge.domain.GameResult;
+import bridge.domain.MovingCommand;
+import bridge.domain.RoundStatus;
+import bridge.repository.GameRepository;
+import bridge.util.OneNumberGenerator;
+import bridge.util.ZeroNumberGenerator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BridgeGameTest {
+
+    private GameRepository gameRepository;
+    private BridgeMap bridgeMap;
+    private BridgeMaker bridgeMaker;
+    private BridgeGame bridgeGame;
+
+    @BeforeEach
+    void setUp() {
+        gameRepository = new GameRepository();
+        bridgeMap = new BridgeMap();
+    }
+
+    @Test
+    @DisplayName("ZeroNumberGenerator를 사용하여 다리를 생성하고 저장하는지 테스트")
+    void makeBridgeWithZeroNumberGeneratorShouldCreateAndSaveBridge() {
+        // Given
+        BridgeNumberGenerator zeroGenerator = new ZeroNumberGenerator();
```

Comment:
저도 이번에 알게 된건데 BridgeNumberGenerator가 `@FunctionalInterface` 어노테이션 달려있는 모습을 확인 할 수 있습니다. 
함수형 인터페이스는 하나의 추상 메서드만 가지는 인터페이스이기 때문에 간단하게 람다식으로 표현이 가능하다고 하네요 그래서 이렇게 새롭게 테스트를 위한 클래스를 새로 생성하기 보다 아래와 같이 사용하는 방법이 좋을 것 같습니다 !

```suggestion
        BridgeNumberGenerator bridgeNumberGenerator = () -> 0;
```
뭔가 함수형 인터페이스에 대한 자료를 드리고 싶은데 마땅히 상황에 맞는 자료가 있는것 같지는 않아서 따로 드리진 못하겠네요 죄송합니다 😅 함수형 인터페이스에 대해서 한번 공부해보시면 좋을 것 같습니다 !

Reply:
이건 몰랐는데 진짜 좋네요....
어노테이션 확인을 제대로 해야겠어요 ... !!